epictrack-api/src/api/services/user.py

```diff
@@ -26,40 +26,67 @@ def get_all_users():
         """Get all users"""
         users = KeycloakService.get_users()
         for user in users:
-            user['group'] = None
+            user["group"] = None
         groups = UserService.get_groups()
-        groups = sorted(groups, key=lambda x: x['attributes']['level'][0])
+        groups = sorted(groups, key=lambda x: x["attributes"]["level"][0])

```

Comment:
u can use the get_level() method here?
groups = UserService.get_groups()
sorted_groups = sorted(groups, key=get_level)